src/main/resources/anet-schema.yml

```diff
@@ -94,6 +94,17 @@ $defs:
         type: string
       componentClass:
         type: string
+      filters:
+        type: array
+        uniqueItems: true
+        items:
+          type: object
+          title: filterType
+          properties:
+            filterField:
+              type: string
+            filterValue:
+              type: array
```

Comment:
The `type` of the `filterValue` actually depends on what the server-side expects (which we can't really check in the schema, that would make it very complex), so something like `type: [array, string, number, boolean]` which probably covers most of the available choices seems a safer choice.